tests/test_bioimg.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+
+from tiledb.cloud.bioimg.helpers import validate_io_paths
+
+
+class BioimgTest(unittest.TestCase):
+    def setUp(self) -> None:
+        self.out_path = "out"
+        self.accepted_pairs = {
```

Comment:
These look like they can just be variables in the `test_validate_io_paths` function, avoiding the need for a separate `setUp` function.